database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -18,8 +18,8 @@ public function up()
             $table->string('name');
             $table->string('email')->unique();
             $table->string('phone')->nullable();
-            $table->string('role');
-            $table->boolean('is_banned')->default(0);
+            $table->string('role')->default(User::DEFAULT_ROLE)->comment('Default role is USER');
```

Comment:
Rather than having an extra comment here, you can just use `->default(User::ROLE_GENERAL_USER)`. So, you don't even need the `DEFAULT_ROLE` property in the User Model. Because if you later change the value of `DEFAULT_ROLE=self::ROLE_LIBRARIAN`, your `comment` here will be misleading.